src/gates/GlobalEventsGate.js

```diff
@@ -0,0 +1,89 @@
+import { useEffect } from 'react';
```

Comment:
Loan events are not global - allowing you to navigate away after creating a loan (and coming back), and it will retain the state + update the balances.